### PR TITLE
ref(discover) Update chart tooltips round 2

### PR DESCRIPTION
**Pull request by markstory**
The last round of tooltip updates were not well loved. These changes put the tooltip content *back* into a tooltip. They also make the tooltip better looking and snappier.

I looked at events v1, metric alerts and dashboards and none of them seemed negatively impacted. Events v1 does reflect the updated tooltips.

Screen Shot 2020-02-14 at 4 11 49 PM
Screen Shot 2020-02-14 at 4 12 03 PM

### CODE REVIEW

**Review by billyvg — APPROVED** [review]
Looks great, should we apply this to all of our charts?

File: src/sentry/static/sentry/app/components/charts/releaseSeries.jsx [hunk 1]
```diff
@@ -98,11 +99,19 @@ class ReleaseSeries extends React.Component {
...
+            const time = moment
               .tz(data.value, utc ? 'UTC' : getUserTimezone())
-              .format('MMM D, YYYY LT')} <br />
-            Release: ${formatVersion(data.name, true)}<br />
-            </div>`;
+              .format('MMM D, YYYY LT');
+            const version = escape(formatVersion(data.name, true));
+            return [
+              '<div class="tooltip-series">',
+              `<div><span class="tooltip-label"><strong>Release</strong></span> ${version}</div>`,
```
Comment (billyvg) [inline src/sentry/static/sentry/app/components/charts/releaseSeries.jsx]: Do we want to `t()` here? Wasn't in the old old.

Comment (markstory) [reply src/sentry/static/sentry/app/components/charts/releaseSeries.jsx]: Yeah I can add that.

**Comment by markstory** [comment]
> Looks great, should we apply this to all of our charts?

I'll see if there is any product interest in extending these tooltips to dashboards and metric alerts.

**Pull request merged**